src/components/assembly_anchor.js

```diff
@@ -0,0 +1,6 @@
+//@flow
+import React from 'react';
+
+export default function AssemblyAnchor({ url }: { url: string }) {
```

Comment:
@kepta The `react-anchorify-text` library is a bit weird in how it handles custom anchors. The `url` prop is passed by `AnchorifyText`. It takes the first child component inside `AnchorifyText` and uses that to build the anchor tags.

[react-anchorify-text/example#L50-L52](https://github.com/georgeOsdDev/react-anchorify-text/blob/develop/example/app.js#L50-L52)